### ISSUE TITLE
kendo mobile and backbone marionette coffeescript eco templates

### ISSUE THREAD

**Issue by phalaphil** [kendo-labs/kendo-backbone#9]
Is there a way to use kendo mobile for replacing the marionette views exclusively, if possible without the need of creating a kendo mobile app? We use the Kendo Mobile widgets, like kendoMobileDrawer, the same way that kendoSplitter is being use in this example : http://www.kendoui.com/blogs/teamblog/posts/12-11-26/backbone_and_kendo_ui_a_beautiful_combination.aspx

Basically, we have the entire Model and data connected to coffeescript and eco templates and only want to use Kendo Mobile for the widgets with our existing backbone + marionette + coffeescript + eco template backend. Any help would be greatly appreciated.

I need to be able to create that widget from inside a backbone view

Basically, https://github.com/kendo-labs/kendo-backbone --> ("Additional work to include integration with Kendo UI Mobile is also in the works.")   I really need this part!!

**Comment by mxriverlynn** [comment]
sadly, this isn't possible at the moment. 

the original work that i had done to get them integrated was thrown out, as it was too complex in how it had to be used, and added little to no benefit. 

i had actually been meaning to remove that info about Kendo UI Mobile integration for some time... but there are some things that are in the works, again... so, while it is technically true that Backbone and Kendo UI mobile may be able to integrate at some point in the future (assuming current plans go through), it's not really possible right now.

**Comment by iampeter** [comment]
@phalaphil I'm really curious what you ended up using, because I'm at a pretty similar point right now.

also, @derickbailey, maybe you could shed some light on this? anything new since 3 months ago?
